### PR TITLE
Merge Bloom Core

### PR DESCRIPTION
**Pull request by ludtkemorgan**
# Pull Request Template

## Issue Overview

This PR addresses #issue

- [x] This change addresses the issue in full
- [ ] This change addresses only certain aspects of the issue
- [ ] This change is a dependency for another issue
- [ ] This change has a dependency from another issue

## Description

This pulls in the latest 16 commits from core. The biggest change is the upgrade to React 18 and also Next 13

## How Can This Be Tested/Reviewed?

Provide instructions so we can review.

Describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration.

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have added QA notes to the issue with applicable URLs
- [ ] I have performed a self-review of my own code
- [ ] I have reviewed the changes in a desktop view
- [ ] I have reviewed the changes in a mobile view
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have assigned reviewers
- [ ] I have run `yarn generate:client` and/or created a migration if I made backend changes that require them
- [ ] I have exported any new pieces added to ui-components
- [ ] My commit message(s) is/are polished, and any breaking changes are indicated in the message and are well-described
- [ ] Commits made across packages purposefully have the same commit message/version change, else are separated into different commits

## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.

### CODE REVIEW

**Comment by ludtkemorgan** [comment]
I'm investigating the broken partner unit test. It passes locally, but not on github actions

**Comment by lisunshiny** [comment]
@ludtkemorgan  qq is it normal that `yarn`ing results in a lockfile?

**Comment by ludtkemorgan** [comment]
> @ludtkemorgan qq is it normal that `yarn`ing results in lockfile changes?

Yes, when running `yarn` it uses the lockfile unless something on the package.json file has changed. In that case it also updates the lockfile. This is only done locally as theoretically it should be updated with the commit

**Comment by lisunshiny** [comment]
hey @ludtkemorgan ! some heads up and some qs:

1) I did some testing on public site and partners portal, some stuff is broken on the public site but I don't think its related to this PR so that overall seems fine
2) I tried doing some cloudinary uploads, which involved logging in as an admin which I assume both tests the new auth cookies and the shared-services stuff.

Qs for you:
* Did you run the integration tests on the public and partners site? We don't have them in our github actions and I still can't get a clean run locally. Its possible that they are failing on main rn but curious to see what the output is regardless
* Is there anything you need from us before taking the PR out of draft and ready for review/merge?

**Comment by ludtkemorgan** [comment]
@lisunshiny I did run the integration tests. The public ones work, but the partner fails for reason unrelated to these changes. Mostly around how we expect the "San Jose" jurisdiction for a few tests that doesn't exist in the doorway seeded data.

Nothing is preventing this from going out of Draft. I'll change that now!

**Comment by ludtkemorgan** [comment]
@lisunshiny logging into partners tests the cookies. It should also work on multiple tabs, as in logging in and then opening a new tab of the app and you should still be logged in.

**Review by ifranch — APPROVED** [review]

File: backend/core/types/src/backend-swagger.ts [hunk 1]
```diff
@@ -3893,6 +3900,12 @@ export interface User {
 
   /**  */
   hitConfirmationURL?: Date
+
+  /**  */
+  activeAccessToken?: string
```
Comment (chriscasto) [inline backend/core/types/src/backend-swagger.ts:3905]: @ludtkemorgan When I hit `/listings` and view the items under `leasingAgents`, I see this property show up along with some other potentially sensitive information.  My understanding of what this represents is limited to what I'm able to grok in bits and pieces in this PR, but at first look it appears as though I would be able to grab the current access and refresh tokens of a leasing agent if they are currently logged in.  Is that accurate?

Comment (chriscasto) [reply backend/core/types/src/backend-swagger.ts:3905]: image

Comment (chriscasto) [reply backend/core/types/src/backend-swagger.ts:3905]: Confirmed:

image

This is a hard block until it's fixed on the Bloom side, IMO

Comment (ludtkemorgan) [reply backend/core/types/src/backend-swagger.ts:3905]: That is a good call out. I brought it up with the team today and we are going to put up another commit to core to remove the sensitive fields. There is a chance we can remove a majority if not all of the leasingAgent fields from the /listings endpoint

Comment (chriscasto) [reply backend/core/types/src/backend-swagger.ts:3905]: Thanks for fixing and merging so quickly!

**Pull request merged**